sdk/python/feast/infra/online_stores/contrib/hbase_online_store/hbase.py

```diff
@@ -0,0 +1,232 @@
+# Created by aurobindo.m on 18/04/22
```

Comment:
want to remove these comments from the files?

Reply:
Done!